app/helpers/collection_helper.rb

```diff
@@ -27,7 +27,8 @@ def collections_search_params
   def collection_breadcrumb_value(collection_id)
     if @response.documents.first&.index_parent_collections.present?
       collection = @response.documents.first.index_parent_collections.find do |coll|
-        coll[:id] == collection_id
+        # re-add leading 'a' to collection catkeys since that's how we index it
+        coll[:id] == collection_id.sub(/^(\d+)$/, 'a\1')
```

Comment:
My fault. I think I got this backwards. And should get the test passing.

```suggestion
        # removing leading 'a' from the collection_id passed to the helper because
        # we remove it from the ids in index_parent_collections
        coll[:id] == collection_id.sub(/^a(\d+)$/, '\1')
```